app/experimenter/settings.py

```diff
@@ -136,6 +136,7 @@
 
 OPENIDC_EMAIL_HEADER = config("OPENIDC_HEADER")
 OPENIDC_AUTH_WHITELIST = (
+    "analysis-results-data",
```

Comment:
Wait why are we adding this here?  This would mean unauthenticated clients can all this API, but we should only need to call this from teh frontend, which is authenticated, right?  If we expose this here than non LDAP could call it and get potentially sensitive data?